src/main/groovy/org/cassalog/core/CreateKeyspace.groovy

```diff
@@ -72,7 +72,7 @@ CREATE KEYSPACE $name WITH replication = { 'class': '${replication.strategy}', '
 
   void replication(Closure closure) {
     closure.delegate = replication
-    closure.resolveStrategy = Closure.DELEGATE_ONLY
+    closure.resolveStrategy = Closure.OWNER_FIRST
```

Comment:
Can you explain why the change to OWNER_FIRST? I am just curious because I am not writing a whole lot of Groovy these days, particularly meta programming :)